browser/css/jsdialogs.css

```diff
@@ -2214,6 +2238,7 @@ kbd,
 
 #ServerAuditDialog .ui-treeview {
 	border-color: transparent;
+	max-width: fit-content;
```

Comment:
If previous comments I added we don't need this.

Reply:
Removed max-width, but added max-height to max-content as the max-height was capped at 150px 

```

.jsdialog:not(.sidebar).ui-treeview {
   max-height: 150px; 
}
```